lib/iris/plot.py

```diff
@@ -652,7 +652,7 @@ def _get_plot_objects(args):
         u_object, v_object = args[:2]
         u, v = _uv_from_u_object_v_object(u_object, v_object)
         args = args[2:]
-        if len(u) != len(v):
```

Comment:
Test breaks without this and doesn't with, which is good.

Also got a couple of image tests break when I checked out the version of this on `main` instead of your version though - any idea why?

Reply:
Seems likely I hadn't updated main recently enough - I don't see these failures now